linux_os/guide/system/network/network-nftables/service_nftables_enabled/rule.yml

```
@@ -38,7 +36,7 @@ ocil: |-
 fixtext: |-
     {{{ fixtext_service_enabled("nftables") }}}
 
-platform: machine
+platform: machine and package[nftables] and not package[firewalld]
```

I think it is good to include `and package[nftables]` but it not necessary to include `and not package[firewalld]`

---

Same motivation as for default_deny_policy rule

---

This combination of platforms is not flexible and will restrict too much this rule.
The simplest way would be only:
`platform: machine and package[nftables]`.

If we want to include `firewalld` in these criteria, the criterion must be the state of the `service` and not the state of the package. Something like this would make more sense and not overly limit the rule:
`machine and package[nftables] not service_disabled[firewalld]`

This is just an example. Currently we don't have the `service_disabled[firewalld]` applicability check in the project but it would be interesting to include it since there is this demand.

If you agree, my suggestion would be to use the simplest form of the `platform: ` line for now so we can move forward with this PR considering only the `nftables` package state. In a separate PR it is possible to include a new applicability check for service state and update these rules. What do you think?

```suggestion
platform: machine and package[nftables]
```